src/routes.tsx

```diff
@@ -3,13 +3,7 @@ import { Home } from '@/page/home/home'
 
 export enum routes {
   home = '/',
-  catalog = '/catalog',
-  sharing = '/sharing',
-  knowledgeBuilder = '/knowledge-builder',
-  dataspaceBuilder = '/dataspace-builder',
-  appBuilder = '/app-builder',
-  documentation = '/documentation',
-  help = '/help'
+  catalog = '/catalog'
```

Comment:
@lolottetheclash 
The new route is `/dataverse`
Could you make the mods please?
Thx 🙏